src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenLocalFunctionTests.cs

```diff
@@ -2242,7 +2242,7 @@ static int B(int val)
     }
     static object D(object val)
     {
-        T Local<T>(T valu) where T : object
+        T Local<T>(T valu) where T : class
```

Comment:
The former is illegal C# code. The diagnostics started showing up after my change. :)